tla/consensus/ccfraft.tla

```diff
@@ -929,9 +1030,13 @@ HandleAppendEntriesResponse(i, j, m) ==
            \* "If AppendEntries fails because of log inconsistency: decrement nextIndex (aka sentIndex +1) and retry"
           /\ UNCHANGED matchIndex
     /\ Discard(m)
-    /\ UNCHANGED <<reconfigurationVars, serverVars, candidateVars, logVars, membershipState>>
+    /\ UNCHANGED <<reconfigurationVars, serverVars, candidateVars, logVars>>
 
-\* Any RPC with a newer term causes the recipient to advance its term first.
+\* Any message with a newer term causes the recipient to advance its term first.
+\* Note that UpdateTerm does not discard message m from the set of messages so this 
+\* message can parsed again by receiver. Note that all other message parsing actions should
```

Comment:
can *be* parsed again *the* receiver.

Reply:
Fixed in https://github.com/microsoft/CCF/pull/5919/commits/9c7329ea95d3f881154abd93a278fe2dad8338d9